javascript-methods/main.js

```diff
@@ -0,0 +1,68 @@
+let x = 5;
+let y = 8;
+let z = 12;
+let maximumValue = Math.max(x, y, z);
+console.log(maximumValue);
+
+const heroes = [
+  'Batman',
+  'Superman',
+  'Spiderman',
+  'Ironman',
+];
+
+let randomNumber = Math.random();
+randomNumber = randomNumber * heroes.length;
+
+let randomIndex = Math.floor(randomNumber);
+console.log(randomIndex);
+
+let randomHero = heroes[randomIndex];
+console.log(randomHero)
+
+
+let library = [
+  {
+    title: 'Book1',
+    author: 'Ted',
+  },
+  {
+    title: 'Book2',
+    author: 'John',
+  },
+  {
+    title: 'Book3',
+    author: 'Fred',
+  },
+];
+
+let lastBook = library.pop();
+console.log(lastBook);
+
+let firstBook = library.shift();
+console.log(firstBook);
+
+var js = {
```

Comment:
We don't want to use the `var` keyword anymore, it's outdated. We should only be using `const` or `let` moving forward.

This should be `const`